cb_read() fails if called in github actions

Hi!
We have set up a continuous benchmark for a package ({gtsummary}).

We want to fetch, read, and report the data from benchmarks with an RMD report.

The code:
```{r bench_fetch, echo=TRUE, message=TRUE, warning=TRUE}
bench::cb_fetch()
```
```{r bench_read}
stored_benchmarks <- bench::cb_read()
```

fails and throws the following error:

label: bench_read
Quitting from lines 21-23 (benchmark2html.Rmd) 
Error: Error in strsplit(x$parent_hashes, " ") : non-character argument

Calls: <Anonymous> ... withVisible -> eval -> eval -> <Anonymous> -> strsplit
In addition: Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
  incomplete final line found by readTableHeader on 'git log --notes=benchmarks --pretty=format:"%H|%h|%P|%N|%s|%D"'
Execution halted
Error: Process completed with exit code 1.

(lines 21-23 correspond to the call to cb_read() )

You can see the workflow here:
https://github.com/oranwutang/gtsummary/runs/2008024051?check_suite_focus=true

The benchmark yaml is here:
https://github.com/oranwutang/gtsummary/blob/benchmark/.github/workflows/continuous-benchmarks.yaml

And the RMD code is here:
https://github.com/oranwutang/gtsummary/blob/benchmark/bench/benchmark2html.Rmd


The code works perfectly on desktop, it only fails while running in github actions workflow.

Would you please point us to a solution to this problem?

Thanks in advance!

--- a/R/continuous_benchmarking.R
+++ b/R/continuous_benchmarking.R
@@ -215,7 +215,7 @@ cb_read <- function(path = ".", additional_columns = NULL) {
   x$benchmarks <- lapply(x$benchmarks, cb_read_benchmark)
 
   # Split the parents into character list-cols
-  x$parent_hashes <- strsplit(x$parent_hashes, " ")
+  x$parent_hashes <- strsplit(as.character(x$parent_hashes), " ")
 
   x$ref_names <- parse_ref_names(x$ref_names)
 

